examples/E03/run_multi.C

```diff
@@ -0,0 +1,44 @@
+//------------------------------------------------
+// The Virtual Monte Carlo examples
+// Copyright (C) 2007 - 2014 Ivana Hrivnacova
+// All rights reserved.
+//
+// For the licensing terms see geant4_vmc/LICENSE.
+// Contact: root-vmc@cern.ch
+//-------------------------------------------------
+
+/// \ingroup E03
+/// \file E03/run_g3.C
+/// \brief Macro for running Example03 with mixed GEANT3 and GEANT4
+
+#include "set_vis.C"
+
+void run_multi(const TString& configMacro1 = "g3tgeoConfig.C", const TString& configMacro2 = "g4tgeoConfig4.C")
+{
+/// Macro function for running Example03 with Geant3 from
+/// Root interactive session
+/// Note that since Root 6 the libraries have to be loaded first
+/// via load_g3.C.
+/// \param configMacro configuration macro name, default \ref E03/g3Config.C
+
+#ifdef G4MULTITHREADED
```

Comment:
This test is too restrictive, as the VMC application can be run in sequential mode with G4 libraries built with MT mode; to do this, there is a parameter in TG4RunConfiguration constructor; it is used for example in O2; see O2/Detectors/gconfig/g4Config.C
So we can better add a dedicated g4tgeoConfig.C here with an explicit setting of the mtMode option.